client-server/src/main/java/Message.java

```diff
@@ -0,0 +1,39 @@
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Message implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    transient public static final String EXIT = "#exit#";
+
+    private final int user_id;
```

Comment:
You don't really need to transfer `user_id` from client to server, because user already knows his id as well as server.

Reply:
done